### ISSUE TITLE
Not compatible with an Android library project

### ISSUE THREAD

**Issue by jkasten2** [OneSignal/OneSignal-Gradle-Plugin#18]
The following error is thrown when used with a Android library project.
```
Could not find property 'applicationVariants' on com.android.build.gradle.LibraryExtension
```

Originally reported in https://github.com/OneSignal/OneSignal-Android-SDK/issues/409

**Comment by leduchuy89vn** [comment]
Any hase solution for this bug?

**Comment by ron-k** [comment]
Encountered same issue. For now chose to not include the plugin in the library, but instead integrate it  (e.g. add the dependencies) manually

**Comment by jkasten2** [comment]
Fixed in PR above and merged into master, will be a part of the next release

**Comment by jkasten2** [comment]
@leduchuy89vn @ron-k This fix is now available in the [0.8.2 Release](https://github.com/OneSignal/OneSignal-Gradle-Plugin/releases/tag/0.8.2)
